sipXyealink/etc/yealink/types.xml

```diff
@@ -200,7 +201,7 @@
 </type>
 <type id="InfoType_type">
     <enum>
-	<option><label>Disabled</label><value>0</value></option>
+	<!--<option><label>Disabled</label><value>0</value></option>-->
```

Comment:
OK, but types.xml is shared with other models as well, does this mean that Disabled option will be removed from the other models as well? If so, what would happen if a version 7 phone already has this option set on Disabled?